### ISSUE TITLE
Custom compiling Makie on PackageCompiler master fails

### ISSUE THREAD

**Issue by asinghvi17** [JuliaLang/PackageCompiler.jl#220]
Running MakieGallery while using PackageCompiler gives this error:

```julia
Running Tutorial simple scatter
(index 1)
┌ Warning: Could not find font Dejavu Sans, using Dejavu Sans
└ @ AbstractPlotting /Users/anshul/.julia/packages/AbstractPlotting/tsTkU/src/conversions.jl:543
Example Tutorial simple scatter failed with source:
    using AbstractPlotting

         x = rand(10)
         y = rand(10)
         colors = rand(10)
         scene = scatter(x, y, color = colors)


ERROR: LoadError: LoadError: LoadError: recursion, font path seems to not contain dejavu sans: /Users/anshul/.julia/packages/AbstractPlotting/tsTkU/src/../assets/fonts
```

no matter whether I use Pkg's inbuilt test mechanism (in the snoopfile) or manually iterate through the Gallery.  

I'm on Makie, AbstractPlotting, GLMakie and PlotUtils master.

However, `]test Makie` runs fine; as does running the examples in the REPL.

The font `Dejavu Sans` is installed on my system.

Compiling Makie using the latest tagged version doesn't throw this error; this isn't by any means a breaking bug, but is probably something to consider.

**Comment by asinghvi17** [comment]
``` 
anshul   ~  julia '/Volumes/Swap/git/cw/misc/Julia/PackageCompiler-scripts/pkgtesttests.jl'                                       5.52G   14:34:07 
Running Tutorial simple scatter
(index 1)
Running Tutorial markersize
(index 2)
Running Tutorial simple line
(index 3)
Running Tutorial adding to a scene
(index 4)
Running Tutorial adjusting scene limits
(index 5)
Running Tutorial basic theming
(index 6)
Running Tutorial heatmap
(index 7)
[...]
```
So it works from the console, just not in PackageCompiler.

**Comment by asinghvi17** [comment]
For reference, here is the code:
```julia
using Makie, StatsMakie
using MakieGallery, RDatasets

 # Load the database
database = MakieGallery.load_database()

# iterate over database
for i in 1:length(database)

   print("Running " * database[i].title * "\n(index $i)\n")

   try

   MakieGallery.eval_example(database[i]);  # evaluate the entry

    catch e

        @warn("Failed, caught")
        print(e)

    end

end
```

**Comment by asinghvi17** [comment]
This issue occurs on both FreeType v2.1.1, FreeTypeAbstraction v0.4.2 and masterbranches.
